### PR TITLE
feat: cySync graphics update

### PR DESCRIPTION
**Pull request by jhasuraj01**
- [x] Wallet Recovery
- [x] Create Wallet
- [x] Receive Dialog

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6306 On cysync graphics are missing for device and cards](https://app.clickup.com/t/9002019994/PRF-6306)

File: packages/cysync-core/src/dialogs/GuidedFlow/context/index.tsx [hunk 1]
```diff
@@ -35,6 +30,19 @@ import {
...
+  EnterWalletNameDeviceGraphics,
+  ConfirmWalletNameDeviceGraphics,
+  ConfirmCreatePinDeviceGraphics,
+  EnterPinDeviceGraphics,
+  VerifyPinDeviceGraphics,
+  GenerateNewWalletDeviceGraphics,
+  EnterSeedphraseDeviceGraphics,
+  SelectSeedphraseWordCountDeviceGraphics,
+  VerifySeedphraseDeviceGraphics,
+} from '@cypherock/cysync-ui/src';
```
Comment (irshadCypherock) [inline packages/cysync-core/src/dialogs/GuidedFlow/context/index.tsx]: Do not import from `src`

**Review by irshadCypherock — CHANGES_REQUESTED** [review]
Please fix the failing tests on CI

File: packages/cysync-core/src/dialogs/GuidedFlow/context/index.tsx (hunk 1 above)
Comment (jhasuraj01) [reply packages/cysync-core/src/dialogs/GuidedFlow/context/index.tsx]: fixed

**Review by irshadCypherock — APPROVED** [review]

**Pull request merged**